XMonad/Util/Font.hs

```diff
@@ -150,9 +159,9 @@ textExtentsXMF (Core fs) s = do
   let (_,a,d,_) = textExtents fs s
   return (a,d)
 #ifdef XFT
-textExtentsXMF (Xft xftfont) _ = io $ do
-  ascent  <- fi <$> xftfont_ascent  xftfont
-  descent <- fi <$> xftfont_descent xftfont
+textExtentsXMF (Xft xftfonts) _ = io $ do
+  ascent  <- fi <$> xftfont_max_ascent xftfonts
+  descent <- fi <$> xftfont_min_descent xftfonts
```

Comment:
Actually, isn't the descent also always positive?  Wouldn't we want the maximal descent here so that we can nicely center text in e.g. the prompt (indeed, I just tried that and it works)?

Reply:
140a6d27a07a8a5bcb3eec24359ffb5b9ce37665